### DISCUSSION TITLE
Adding non-Twilio verified number as Caller ID in Dialpad

### DISCUSSION THREAD

**Discussion by TarikCarvalho08** [twilio-professional-services/flex-project-template#361] [Q&A] [answered]
Hi,

Can someone guide me on how I can get a non-Twilio number to display inside the Dialpad, on the Caller ID dropdown menu? At the moment I only can see purchased Twilio numbers on the account. I already changed the Caller ID to the verified non-Twilio number in the Voice configuration settings of Flex. I added the number as the caller ID and enabled the Dialpad in Flex.

I tried as well adding to the property ("include_outgoing_only_numbers": true) into the Caller ID attribute, but still nothing changes.

Thank you,

Tarik Carvalho.

**Comment by dremin** [comment, marked as answer]
Hi @TarikCarvalho08! Apologies for the delay here! The latest versions of the template enable showing verified caller IDs in the dropdown by default. You may follow [these instructions](https://twilio-professional-services.github.io/flex-project-template/building/merge-future-updates) to update your repository with the latest template updates, then perform a deployment. Also, please make sure the caller IDs are set up within Twilio Console > Phone Numbers > Manage > Verified Caller IDs.
